templates/hiera.yaml.epp

```diff
@@ -61,7 +61,15 @@ hierarchy:
 <% if $hr['options'] { -%>
     options:
     <%- $hr['options'].each |$key,$value| { -%>
+    <%- if $value =~ Array { -%>
+      <%= $key %>:
+        <%- $value.each |$value| { -%>
+        - "<%= $value %>"
+        <%- } -%>
+    <%- } -%>
+    <%- unless $value =~ Array { -%>
```

Comment:
I think this should be part of an `else` clause, as you want one or the other.

Reply:
ok, done